src/mocks/daily-intake.ts

```diff
@@ -1,8 +1,11 @@
+import uuid from 'uuid';
+
 import { IDailyIntake } from '@/models/nutrition';
 export const dailyIntake = (day: string): IDailyIntake => {
+    console.log('Mock for daily intake,', day);
```

Comment:
`console.log` ;)

Reply:
Here I left it with conciously to know in console which day mock refers to - it is just a mock data